atd-vze/src/views/Crashes/recommendationsDataMap.js

```diff
@@ -0,0 +1,20 @@
+export const recommendationsDataMap = [
```

Comment:
unless you're planning to expand this and want to `map()` over it, it's fine to make this an object instead of an array.